src/model_theory/satisfiability.lean

```diff
@@ -186,6 +188,64 @@ begin
     rw [← lift_le.{_ (max (max u w') v w)}, lift_lift, lift_lift, lift_lift, lift_lift] }
 end
 
+theorem exists_elementarily_equivalent_card_eq_of_le (M : Type w') [L.Structure M] [nonempty M]
```

Comment:
Please provide a docstring for theorems that are more than a couple lines long. In this case theorem, please also mention explicitly the relevance of universes (i.e. you get a model in universe `w`).

Reply:
I've clarified that this is an alternate version of DLS, and the main reason you'd use this one would be because of universes.